scrnaseq.Rmd

```diff
@@ -2290,91 +2369,70 @@ for (i in names(genesets_to_export)) {
-                      enriched_pathways=enriched)
+                      assay=DefaultAssay(sc),
+                      delayed_array=FALSE)
 ```
 
-## Output files
-
-All files generated with this report are written into the provided output folder `r param$path_out`: 
-
-* Annotation files  
-  + `r basename(param$file_annot)`: Table that contains several identifiers and annotation (columns) per gene (rows) (optional)   
-* Differentially expressed genes  
-  + degs_cluster_vs_rest.xlsx: Excel file with one tab per cell cluster  
-* Functional enrichment of differentially expressed genes per cell cluster  
-  + Functions_DEG_down_cluster_1.xlsx, Functions_DEG_up_cluster_1.xlsx, ...: Excel files with one tab per database
-* Loupe Cell Browser files  
-  + Seurat2Loupe_umap.csv: Seurat UMAP 2D visualisation   
-    Import to Loupe through "Import Projection"   
-  + Seurat2Loupe_metadata.csv: Seurat categorial meta data including clusters and cell cycle phases   
-    Import to Loupe through "Import Categories"   
-  + Seurat2Loupe_genesets.csv: Seurat differentially expressed genes   
-    Import to Loupe through "Import Lists"   
-* Cerebro files:
-  + cerebro.crb: Can be loaded into the Cerebro Browser  
-  
-```{r save}
-# Add colors to the sc object for compatibility with downstream tools
-# List names should correspond to meta-data columns
-# This could be done for other meta-data as well 
-sc@misc$colors = list("orig.ident" = param$col_samples,
-                      "seurat_clusters" = param$col_clusters)
-
-save.image(file=paste0(param$path_out, "/scrnaseq.RData"))
-```
+* list files here
+* shortly explain howto
 
 # Parameters and software versions
 The following parameters were used to run the workflow.  
 ```{r parameters_table}
 out = scrnaseq_params_info(params=param)
 
 knitr::kable(out, align="l") %>% 
-  kableExtra::kable_styling(bootstrap_options = c("striped", "hover"), full_width=FALSE, position="left")
+  kableExtra::kable_styling(bootstrap_options=c("striped", "hover"), full_width=FALSE, position="left")
 ```
 
-This report was generated using the [scrnaseq](https://github.com/ktrns/scrnaseq) GitHub repository. Software versions were collected at run time. 
+This report was created with generated using the [scrnaseq](https://github.com/ktrns/scrnaseq) GitHub repository. Software versions were collected at run time. 
 ```{r versions, message=FALSE}
 out = scrnaseq_session_info(param$path_to_git)
 
 knitr::kable(out, align="l") %>% 
-  kableExtra::kable_styling(bootstrap_options = c("striped", "hover"))
+  kableExtra::kable_styling(bootstrap_options=c("striped", "hover"))
 ```
 
+
 # References
 ```{r bib, message=FALSE}
+# If the DOI publication servers cannot be reached, there will be no citations, knitcitations will not write a references.bib file and pandoc will stop. This makes sure that there is always at least one citation.
+invisible(knitcitations::citep(citation("knitr")))
```

Comment:
I added this very much to the top, so its there before the early exit can happen in case of an error.

Reply:
But I see that you removed the `knit` exit, and added a `stop`, so never mind.

Reply:
no, good idea, it is always better that there is at least one entry - it is annoying if at the end pandoc fails